pkg/endpoints/cluster.go

```diff
@@ -195,13 +195,24 @@ func (r Resource) GetProperties(request *restful.Request, response *restful.Resp
 	isOpenShift := IsOpenShift(r, installedNamespace)
 	isReadOnly := IsReadOnly()
 	pipelineVersion := GetPipelineVersion(r, pipelineNamespace, isOpenShift)
-
 	isTriggersInstalled := IsTriggersInstalled(r, triggersNamespace, isOpenShift)
+
+	properties := Properties{
+		InstallNamespace: installedNamespace,
+		DashboardVersion: dashboardVersion,
+		PipelineVersion:  pipelineVersion,
+		IsOpenShift:      isOpenShift,
+		ReadOnly:         isReadOnly,
+	}
+
+	// If running on OpenShift, set the logout url
+	if isOpenShift {
+		properties.LogoutURL = "/oauth/sign_out"
+	}
+
 	if isTriggersInstalled == false {
```

Comment:
Wrong test 😢

Reply:
If you can try again... sorry for the inconvenience

Reply:
no problem at all 😸